model/app.model.js

```diff
@@ -15,3 +15,11 @@ exports.fetchAllEndpoints = () => {
     })
 }
 
+exports.fetchArticleById = (article_id) => {
+    
+    const query = `SELECT * FROM articles;`
```

Comment:
Can we make the SQL query use the article_id so we are only recieving the article we need from the database.